helm-chart/templates/config-map.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: analyzer-config
+  namespace: default
+data:
+  config.yaml: |
+    ######################
+    # Config for profile #
+    ######################
+
+    override_output_model_repository: True
+    run_config_search_disable: True
+    triton_http_endpoint: localhost:8000
+    triton_grpc_endpoint: localhost:8001
+    triton_metrics_url: http://localhost:8002/metrics
+
+    concurrency: 1,2
+    batch_sizes: 1
+
+    profile_models: 
+      resnet50_libtorch:
```

Comment:
I think it would be better to use the example add sub model so that the users can really get started with helm deployment based on the models currently available to them.